README.md

```diff
@@ -0,0 +1,2 @@
+Hosted on: https://shimels1.github.io/Shimels_and_Wisdom_Colaboration/
```

Comment:
In the readme, section please use a proper temple like Microverse since your readme section does not explain the detail about your project like the About section, get started section, or Authors section...